uaclient/entitlements/tests/test_cis.py

```diff
@@ -12,12 +12,13 @@
 
 @pytest.fixture
 def entitlement(entitlement_factory):
-    return entitlement_factory(
-        CISEntitlement,
-        allow_beta=True,
-        called_name="cis",
-        additional_packages=["pkg1"],
-    )
+    with mock.patch("os.getuid", return_value=0):
```

Comment:
Okay, I think we can make this test a little easier to understand if the `UAConfig` class can accept an `is_root` variable. In that way, we can add a new variable to the `entitlement_factory` and pass it to the `FakeConfig` instance. This will avoid the mocking situation we are seeing on the factory functions

Reply:
Yeah, works well and helped solve some problems in a simple way